explainers/subresource-loading.md

```diff
@@ -156,6 +156,10 @@ subresources, `a.js`, `b.js`, and `c.png`, from the bundle.
 `urn:uuid:f81d4fae-7dec-11d0-a765-00a0c91e6bf6` is also loaded from
 the bundle, and a subframe is instantiated as an
 [opaque origin](https://html.spec.whatwg.org/multipage/origin.html#concept-origin-opaque) frame.
+Even if a custom scheme handler for `urn` schme was registered by
+[registerProtocolHandler(scheme, url)](https://html.spec.whatwg.org/multipage/system-state.html#dom-navigator-registerprotocolhandler)
+method, the browser should ignore the registered handler and load the resource
```

Comment:
Nit; Could we replace "should" with "must"?
(This is an explainer, so this is not an issue, however, "must" may be better for this case, IMO)

Reply:
Done